Cargo.toml

```diff
@@ -20,9 +23,9 @@ serde = { version = "1.0.144", features = ["derive"] }
 serde_repr = "0.1"
 enum-iterator = "1.2.0"
 
-dotenvy = "0.15.5"
 log = "0.4.17"
-env_logger = "0.9.1"
```

Comment:
Wouldn't it cause some issues if one uses the library without a logger, but it tries to log?